<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <include file="$(find gazebo_ros)/launch/empty_world.launch">
    <arg name="world_name" value="$(find beacon_gazebo_sim)/worlds/test.world"/>
<!--    <arg name="gui" value="false"/>-->
<!--    <arg name="headless" value="true"/>-->
  </include>
  <include file="$(find beacon_gazebo_sim)/launch/robot_autonet_v1.launch" >
    <arg name="init_pose" value="-x 0 -y 0 -z 0.2" />
    <arg name="robot_name" value="Robot1" />
  </include>
  <include file="$(find beacon_gazebo_sim)/launch/robot_vc_robot_v1.launch" >
    <arg name="init_pose" value="-x 2 -y 2 -z 0.2" />
    <arg name="robot_name" value="Robot2" />
  </include>
  <include file="$(find beacon_gazebo_sim)/launch/beacon.launch" >
    <arg name="init_pose" value="-x 1 -y 1 -z 0.2" />
    <arg name="beacon_name" value="beacon_1" />
  </include>
  <include file="$(find beacon_gazebo_sim)/launch/beacon.launch" >
    <arg name="init_pose" value="-x 0 -y 4 -z 0.2" />
    <arg name="beacon_name" value="beacon_2" />
  </include>
  <include file="$(find beacon_gazebo_sim)/launch/beacon.launch" >
    <arg name="init_pose" value="-x -1 -y -1 -z 0.2" />
    <arg name="beacon_name" value="beacon_3" />
  </include>
  <include file="$(find beacon_gazebo_sim)/launch/beacon.launch" >
    <arg name="init_pose" value="-x 3 -y -3 -z 0.2" />
    <arg name="beacon_name" value="beacon_4" />
  </include>
  <include file="$(find beacon_gazebo_sim)/launch/beacon.launch" >
    <arg name="init_pose" value="-x 4 -y -3 -z 0.2" />
    <arg name="beacon_name" value="beacon_5" />
  </include>
  <include file="$(find beacon_gazebo_sim)/launch/beacon.launch" >
    <arg name="init_pose" value="-x 5 -y -3 -z 0.2" />
    <arg name="beacon_name" value="beacon_6" />
  </include>
  <include file="$(find beacon_gazebo_sim)/launch/beacon.launch" >
    <arg name="init_pose" value="-x 6 -y -3 -z 0.2" />
    <arg name="beacon_name" value="beacon_7" />
  </include>
  <include file="$(find beacon_gazebo_sim)/launch/beacon.launch" >
    <arg name="init_pose" value="-x 6 -y -2 -z 0.2" />
    <arg name="beacon_name" value="beacon_8" />
  </include>
  <include file="$(find beacon_gazebo_sim)/launch/beacon.launch" >
    <arg name="init_pose" value="-x 6 -y -1 -z 0.2" />
    <arg name="beacon_name" value="beacon_9" />
  </include>
  <include file="$(find beacon_gazebo_sim)/launch/beacon.launch" >
    <arg name="init_pose" value="-x 6 -y 0 -z 0.2" />
    <arg name="beacon_name" value="beacon_10" />
  </include>
</launch>